### ISSUE TITLE
Trying to download mentions_count as a version

### ISSUE THREAD

**Issue by JeremieDoctrine** [terraform-linters/tflint#2175]
### Summary

We are running tflint on many of our CI's

From time to time it fails installing with the following error:

```
====================================================
Looking up the latest version ...
Downloading TFLint mentions_count
curl: (22) The requested URL returned error: 404 

Exited with code exit status 22
```

We are installing tflint this way:
```
curl  --retry 3 -s https://raw.githubusercontent.com/terraform-linters/tflint/master/install_linux.sh | TFLINT_VERSION=latest bash
```

My guess is that this grep https://github.com/terraform-linters/tflint/blob/master/install_linux.sh#L46 is not catching the tag_name but another row. Not exactly sure why at the moment. 

**Downloading TFLint mentions_count** seems totally wrong.

### Command

TFLINT_VERSION=latest bash

### Terraform Configuration

```terraform
n/a
```

### TFLint Configuration

```HCL
n/a
```

### Output

```Shell
====================================================
Looking up the latest version ...
Downloading TFLint mentions_count
curl: (22) The requested URL returned error: 404 

Exited with code exit status 22
```
```


### TFLint Version

0.54.0

### Terraform Version

1.9

### Operating System

- [X] Linux
- [ ] macOS
- [ ] Windows

**Comment by bendrucker** [comment]
This is really odd, surely there is some pattern? A specific release or version of the tools or something? I've tried querying the Github API and doing different things like deleting `tag_name` from the response and can't reproduce.

**Comment by JeremieDoctrine** [comment]
We haven't found any pattern and it's indeed very odd ... we are installing it the same way on all of our CI's.

One thing I was thinking about was about 403 ...

**Comment by bendrucker** [comment]
Ah on a second pass I see how this is happening. It's no coincidence that `mentions_count` is the last key in the object.

This does reproduce:

```sh
curl --fail -Ss https://api.github.com/repos/terraform-linters/tflint/releases/latest | jq --compact-output | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'
```

```
mentions_count
```

The script is newline dependent. Without newlines, `grep` just matches the single line of output and `sed` grabs the last value. And `jq --compact-output` reproduces that. 

Why this would be non-deterministic assuming it comes from GitHub is a mystery. But the script could definitely be written more robustly to not be newline-dependent. I'll make that change.
